dffml/source/csv.py

```diff
@@ -121,12 +121,12 @@ class CSVSource(FileSource, MemorySource):
                     del row[header]
             # Set the features
             features = {}
-            for key, value in row.items():
-                if value != "":
```

Comment:
`key` renamed to `_key` because we are already using `key` which gets overwritten. For consistency `value` changed to `_value`